salt/annotations/init.sls

```diff
@@ -50,7 +50,11 @@ var-directory:
             - builder: annotations-repository
 
     cmd.run:
-        - name: chmod -R g+s /srv/annotations/var
+        - name: |
+            chmod -R g+s /srv/annotations/var
+            mkdir -p /srv/annotations/var/logs
+            chown elife:www-data /srv/annotations/var/logs
```

Comment:
`/srv/annotations/var/logs` needs to be writable by `www-data`. This will progressively get simpler as all containers will run as `www-data` and we won't need the special group permissions to share it between `elife` and `www-data`